EventFilter/CTPPSRawToDigi/interface/CTPPSElectronicIndex.h

```diff
@@ -0,0 +1,13 @@
+#ifndef EventFilter_CTPPSRawToDigi_CTPPSElectronicIndex_H
+#define EventFilter_CTPPSRawToDigi_CTPPSElectronicIndex_H
+
+namespace ctppspixelobjects {
+  struct CTPPSElectronicIndex {
```

Comment:
if we are to follow the pixel tracker code, then this struct should be simply ```ElectronicIndex```.
Is there enough common/templated code with the pixel tracker to justify this?
From the context, it looks like CTPPSPixelElectronicIndex without a namespace is enough.

Reply:
Done, but the file name was kept. Do you want to change that as well?

Reply:
Done.